src/SoulMemory/DarkSouls1/PresetFlag.cs

```diff
@@ -0,0 +1,2255 @@
+    Dead_Undead_Asylum_Sword_Black_Knight_1 = 11810851,
+
+    [AnnotationAttribute(Name = "11810900 | Stray Demon")]
+    Stray_Demon = 11810900,
+
+    [AnnotationAttribute(Name = "11815010 | Resting at Firelink Shrine")]
+    Resting_at_Firelink_Shrine_1 = 11815010,
+
+    [AnnotationAttribute(Name = "11815392 | Resting at Firelink Shrine")]
+    Resting_at_Firelink_Shrine_2 = 11815392,
+
+    [AnnotationAttribute(Name = "50000010 | Tiny Being's Ring")]
+    Tiny_Beings_Ring = 50000010,
+
+    [AnnotationAttribute(Name = "50000070 | Speckled Stoneplate Ring (Looted or Given)")]
+    Speckled_Stoneplate_Ring_Looted_or_Given = 50000070,
+
+    [AnnotationAttribute(Name = "50000081 | Oscar, Don't gift Big Pilgrim Key")]
+    Oscar_Dont_gift_Big_Pilgrim_Key = 50000081,
+
+    [AnnotationAttribute(Name = "50000090 | Receive Lordvessel (Gwynevere Alive or Dead)")]
+    Receive_Lordvessel_Gwynevere_Alive_or_Dead = 50000090,
+
+    [AnnotationAttribute(Name = "51020130 | Firelink Shrine, Ring of Sacrifice")]
+    Firelink_Shrine_Ring_of_Sacrifice = 51020130,
+
+    [AnnotationAttribute(Name = "51200200 | Darkroot Basin, Grass Crest Shield")]
+    Darkroot_Basin_Grass_Crest_Shield = 51200200,
+
+    [AnnotationAttribute(Name = "51300020 | Catacombs, Darkmoon Seance Ring")]
+    Catacombs_Darkmoon_Seance_Ring = 51300020,
+
+    [AnnotationAttribute(Name = "51310140 | Tomb of the Giants, Covetous Silver Serpent Ring")]
+    Tomb_of_the_Giants_Covetous_Silver_Serpent_Ring = 51310140,
+
+    [AnnotationAttribute(Name = "51510560 | Anor Londo, Dragon Tooth")]
+    Anor_Londo_Dragon_Tooth = 51510560,
+
+    [AnnotationAttribute(Name = "51510940 | Anor Londo, Blacksmith Giant Hammer")]
+    Anor_Londo_Blacksmith_Giant_Hammer = 51510940,
+
+    [AnnotationAttribute(Name = "51600380 | Valley of the Drakes, Red Tearstone Ring")]
+    Valley_of_the_Drakes_Red_Tearstone_Ring = 51600380,
+
+    [AnnotationAttribute(Name = "51810060 | Northern Undead Asylum, Rusted Iron Ring")]
+    Northern_Undead_Asylum_Rusted_Iron_Ring = 51810060,
+
+    [AnnotationAttribute(Name = "51810080 | Northern Undead Asylum, Peculiar Doll")]
+    Northern_Undead_Asylum_Peculiar_Doll = 51810080,
+
+    [AnnotationAttribute(Name = "61200500 | Darkroot Garden, Crest of Artorias Door")]
+    Darkroot_Garden_Crest_of_Artorias_Door = 61200500,
+
+    [AnnotationAttribute(Name = "61200501 | ? on Touched Sif's Gate")]
+    A_on_Touched_Sifs_Gate_1 = 61200501,
+
+    [AnnotationAttribute(Name = "71010000 | Andre, Talk To For First Time")]
+    Andre_Talk_To_For_First_Time = 71010000,
+
+    [AnnotationAttribute(Name = "71500001 | Crestfallen Merchant, Talk To For First Time")]
+    Crestfallen_Merchant_Talk_To_For_First_Time = 71500001,
+
+    [AnnotationAttribute(Name = "71510000 | Blacksmith Giant, Talk To For First Time")]
+    Blacksmith_Giant_Talk_To_For_First_Time = 71510000,
+
+    [AnnotationAttribute(Name = "103249371 | Set when sitting down on Darkmoon Tomb bonfire 1")]
```

Comment:
Not really... There's some bonfires that are flag as discovered at (almost) random points in the game, so other flags that happen when you sit there for the first time  are really useful.

A pretty well known example is the Anor Londo first bonfire. 
If you use the bonfire autosplit it will split halfway through sen's fortress instead of when you first discover it (or even are in the same map chunk tbh).
So most people just split at the elevator instead.

If these specifically were in kahmul's notes, I'm pretty sure they're what have been or is used in some routes.

Reply:
Ok nvm.. That WAS the idea, but I'm testing these and they don't seem to work... 
I'm just gonna remove a whole bunch of "useless" flags so that it's cleaner anyways...